n2v/methods/rmks.py

```diff
@@ -6,7 +6,521 @@
 
 import numpy as np
 from opt_einsum import contract
-from scipy.optimize import minimize
+import psi4
+import time
 
 class MRKS():
-    pass
\ No newline at end of file
+    """
+    Wavefunction to KS_potential method based on
+    [1] [PRL 115, 083001 (2015)],
+    [2] [J. Chem. Phys. 146, 084103 (2017)].
+
+    The XC potential is calculated on the grid
+    instead of on the potential basis set.
+    Whereas, the guide potential is still used and
+    plays the role of v_hartree.
+    And because of this, the gird for vxc for output
+    has to be specified beforehand.
+    
+    For CIWavefunction as input, make sure to turn on
```

Comment:
Can we add a line that verifies that this is on and raise an exception? Or would psi4 complain automatically?